src/main/java/com/teradata/tpcds/Session.java

```diff
@@ -26,19 +30,20 @@
 import static com.teradata.tpcds.Options.DEFAULT_SEPARATOR;
 import static com.teradata.tpcds.Options.DEFAULT_SUFFIX;
 
-public class Session
+public class Session implements Serializable
```

Comment:
implements should be on a newline to match the style of the rest of the project.  (I thought that would be caught by checkstyle, but I guess not)